.editorconfig

```
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+
+[*.json]
+indent_size = 2
+
+[*.{markdown,md}]
+indent_size = 4
+max_line_length = 80
```

```suggestion
max_line_length = 80
trim_trailing_whitespace = false
```
In Markdown, trailing whitespace is significant. Specifically, two spaces at the end of the line indicate a line break.